zerver/lib/actions.py

```diff
@@ -1415,7 +1415,7 @@ def do_add_subscription(user_profile, stream, no_log=False):
         event = dict(type="subscription", op="peer_add",
                      subscriptions=[stream.name],
                      user_email=user_profile.email)
-        send_event(event, user_ids)
+        send_event(event, active_user_ids(user_profile.realm))
```

Comment:
Yep!

-Tim Abbott (mobile)
On Jun 25, 2016 13:21, "Kartik Maji" <notifications@github.com> wrote:

> In zerver/lib/actions.py
> <https://github.com/zulip/zulip/pull/749#discussion_r68493400>:
>
> > @@ -1415,7 +1415,7 @@ def do_add_subscription(user_profile, stream, no_log=False):
> >          event = dict(type="subscription", op="peer_add",
> >                       subscriptions=[stream.name],
> >                       user_email=user_profile.email)
> > -        send_event(event, user_ids)
> > +        send_event(event, active_user_ids(user_profile.realm))
>
> Cool. There are few more tests that are failing. I will fix those and
> update this PR. Sounds okay?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zulip/zulip/pull/749/files/b8588740a9e6b01d69fc546ff5b65f339c186687..c3e93506eddda5c9ce6b08d918a7a85182ea392b#r68493400>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACnm2pbfb29Pb-doEiDWtxNIXfbS4snHks5qPY2_gaJpZM4IWfUx>
> .
>